src/Rings/mpoly-affine-algebras.jl

```diff
@@ -38,15 +38,20 @@ and $I$ is a homogeneous ideal of $R$ when we grade $R$ according to the corresp
 weighted degree, then $p/q$ represents the Hilbert series of $A$ as a rational function
 with denominator $q = (1-t^{w_1})\cdots (1-t^{w_n})$. 
 
-See also `hilbert_series_reduced(A::MPolyQuo)`.
+See also `hilbert_series_reduced`.
 """
-function hilbert_series(A::MPolyQuo)
+function hilbert_series(A::U) where U <: Union{MPolyRing{T}, MPolyQuo{T}} where T
```

Comment:
The `Union` can be put directly in lieu of `U`, and `T` is not used so can be removed:
```suggestion
function hilbert_series(A::Union{MPolyRing, MPolyQuo})
```